app/notifications/admin/businesses/hiring_invoice_request_notification.rb

```diff
@@ -0,0 +1,22 @@
+module Admin
+  module Businesses
+    class HiringInvoiceRequestNotification < ApplicationNotification
+      deliver_by :database
+      deliver_by :email, mailer: "AdminMailer", method: :businesses_hiring_invoice_request
+
+      param :form
+
+      def title
+        t("notifications.admin.new_hire_form_notification.title", business: form.business.name)
```

Comment:
I think this key should match the class name.

Reply:
fixed in f11aa8c4682b4abe64e688ec622e83002b1177dc